lib/execution_engine2/db/models/transfer/check_jobs_for_finish_times.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+try:
+    from .transfer_ujs_njs import MigrateDatabases
+    from execution_engine2.db.models.models import Status, valid_status
+except:
+    from transfer_ujs_njs import MigrateDatabases
+    from models import *
+
+
+ee2_jobs = MigrateDatabases().ee2_jobs
+
+count = 0
+for job in ee2_jobs.find():
+    count += 1
+    job_id = job["_id"]
+    status = job["status"]
+    try:
+        valid_status(status)
+    except:
+        raise Exception(job_id)
+    if count % 500 == 0:
+        print(f"Processed {count} records")
+
+    if status in [Status.error.value, Status.completed.value, Status.terminated.value]:
+        end_time = job.get("finished")
+        if end_time is None:
+            raise Exception("End time is NONE", job_id)
```

Comment:
do we want to raise here or return all bad job ids and fix them in a batch?

Reply:
I think raise is good so the script crashes and I can just start looking at the database. That's what I did already  and fixed the jobs below

Reply:
👍